picard/__init__.py

```diff
@@ -83,7 +82,11 @@ def crash_handler():
     # with minimum chance to fail.
     from tempfile import NamedTemporaryFile
     import traceback
-    trace = traceback.format_exc()
+    if exc:
+        # Since Python 3.10 calling only traceback.format_exception(exc) is possible
```

Comment:
I think we should test Python version, this is likely to be deprecated at some point, it'll make easier to spot it and remove the need of a comment :

```python
        if sys.version_info < (3, 10):
            trace_list = traceback.format_exception(None, exc, exc.__traceback__)
        else:
            trace_list = traceback.format_exception(exc)
```

Also the first argument is ignored since Python 3.5 (hence None).

By the way, to maintain compatibility they wrote rather hacky code: https://github.com/python/cpython/blob/812245ecce2d8344c3748228047bab456816180a/Lib/traceback.py#L93-L101

Reply:
Done in be432d0701dd7ee993a86dcc0b104a60b6b4147a